ecl2df/common.py

```diff
@@ -293,14 +293,21 @@ def parse_opmio_deckrecord(
                     rec_dict[item_name] = record[item_idx].get_data_list()
                 # (the caller is responsible for unrolling this list with
                 # correct naming of elements)
+
+                # When we parse a list, some values in it can be defaulted, for
+                # which the default values are not provided in the JSON. Return
+                # these defaulted values as NaN's for the calling code to fix.
+                for idx in range(len(record[item_idx])):
+                    if record[item_idx].__defaulted(idx):
```

Comment:
Is double underscore intended and needed here?

Reply:
It is intended, I have not found another way to do it through `opm`s api.

Reply:
Is the `__defaulted` a part of the published api?

Reply:
Not quite sure, but have not gotten any hint to other and better alternatives in https://github.com/OPM/opm-common/issues/2598

Reply:
Ok, then I think that information would be good to add in a comment and/or the commit body.

Reply:
Added in comment.

Reply:
Technically not private but name mangling (though they could have intended it as private :shrug:)